packages/mdc-checkbox/README.md

```diff
@@ -255,8 +255,28 @@ not return an object.
 
 ## Theming
 
-MDC Checkboxes use the theme's primary color by default for checked and indeterminate states, and are completely dark theme
-aware.
+MDC Checkboxes use the theme's secondary color by default for "marked" states (i.e., checked or indeterminate), and are completely dark theme aware.
+
+### Sass Mixins
+
+The following mixins apply only to _enabled_ checkboxes. It is not currently possible to customize the color of a _disabled_ checkbox.
+
+Mixin | Description
+--- | ---
+`mdc-checkbox-stroke-and-container-fill-color($unmarked-stroke-color, $unmarked-container-fill-color, $marked-stroke-color, $marked-container-fill-color, $animate)` | Generates CSS classes to set and animate the stroke color and/or container fill color of a checkbox
+`mdc-checkbox-ink-color($color)` | Sets the ink color of the checked and indeterminate icons
+`mdc-checkbox-focus-indicator-color($color)` | Sets the color of the focus indicator
+`mdc-checkbox-ripple($ripple-config)` | Sets the ripple to the given [ripple configuration][ripple-readme]
+
+[ripple-readme]: https://github.com/material-components/material-components-web/blob/master/packages/mdc-ripple/README.md
+
+#### `mdc-checkbox-stroke-and-container-fill-color($unmarked-stroke-color, $unmarked-container-fill-color, $marked-stroke-color, $marked-container-fill-color, $animate)`
+
+Generates CSS classes to set the stroke color and/or container fill color of a checkbox in its marked and unmarked states.
+
+All parameters are optional, and if left unspecified will use their default values.
+
+If you plan to use CSS-only checkboxes, set `$animate` to `false` to prevent the mixin from generating `@keyframes` and CSS classes used by the JavaScript component.
```

Comment:
I think it should be $keyframeAnimate...not just $animate. Doesn't the CSS-only checkbox still do some animation? Just not with keyframes?

Reply:
Getting super-pedantic here, but would that be transition rather than animation in that case? In which case animate might be enough distinction, but I could go either way.

Reply:
I originally named it `$generate-keyframes`, so I changed it back to that. Does that work?

You're correct - the CSS-only checkbox uses `transition` to tween from marked to unmarked states instead of `animation`. As you suggested, it's probably more accurate to name the parameter something with "keyframes", because that's what it really controls.

Reply:
generate-keyframes does sound better to me.